esrally/mechanic/provisioner.py

```diff
@@ -168,10 +169,10 @@ def __init__(self, cluster_settings, es_installer, plugin_installers, preserve,
         self.telemetry = telemetry
         self.logger = logging.getLogger(__name__)
 
-    def prepare(self, binary):
+    def prepare(self, binaries):
         if not self.preserve:
-            self.logger.info("Rally will delete the benchmark candidate after the benchmark")
-        self.es_installer.install(binary["elasticsearch"])
+            logging.getLogger(__name__).info("Rally will delete the benchmark candidate after the benchmark")
```

Comment:
Any reason not to use `self.logger` here?

Reply:
I'm not really sure why this was changed - I'll change it back!